.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+	"tabWidth": 4,
+	"useTabs": false,
```

Comment:
This config file specifies indentation with 4 spaces, but

* this file uses tabs as indentation
* the other new files use 2 spaces as indentation

Maybe your config does not apply properly?

Reply:
Might be. But there is a separate PR for the config; let's discuss it there. The file is only in this PR until all the other questions are addressed. I will remove it at the end.

Reply:
Alright, now I saw your linked issue about the Prettier config. So I guess you'll reformat the other files before merging, too?